### ISSUE TITLE
Many examples not listed in examples index

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8073]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#442|
|Type|defect|
|Created|2003-12-20 02:34:35Z|

<details><summary>Searchable metadata</summary>

```
trac-id__442 442
type__defect defect
reporter__exarkun exarkun
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__documentation documentation
time__1071887675000000 1071887675000000
changetime__1470345713821356 1470345713821356
version__ 
owner__ 
cc__exarkun cc__hypatia
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
exarkun@boson:~/projects/python/Twisted/doc/examples$ for i in *.py; do if !
grep $i index.xhtml > /dev/null; then echo $i; fi; done
cred.py
echoclient_udp.py
echoserv_udp.py
email.py
filewatch.py
gethostbyname.py
gpsfix.py
insultsexample.py
longex2.py
msn_example.py
oscardemo.py
pyuidemo.py
shaper.py
simpleguardex.py
simpleguardex2.py
soap.py
sshsimpleclient.py
sshsimpleserver.py
testlogging.py
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Should change it so it can be automatically generated. See issues 505 and 506
for that.

However, I'll do this one manually and try and close it before 1.1.2 &amp;mdash; as
long as that occurs 7 days from now or more...
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 3e488bca70159bd8d7f5a62dc55e0ab9879f229b
```
#!CommitTicketReference repository="" revision="3e488bca70159bd8d7f5a62dc55e0ab9879f229b"
Merge pull request #442 from twisted/8705-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8705

Do not use inspect.getargspec() in twisted/python/usage.py on Python 3
```
